extensions/replication/utils/ObjectQueueEntry.js

```diff
@@ -114,16 +98,36 @@ class ObjectQueueEntry extends ObjectMD {
 
     toReplicaEntry(site) {
         const newEntry = this.clone();
-        newEntry.setBucket(this.getReplicationTargetBucket());
-        newEntry.setReplicationSiteStatus(site, 'REPLICA');
-        newEntry.setReplicationStatus('REPLICA');
+        newEntry
+            .setBucket(this.getReplicationTargetBucket())
+            .setReplicationSiteStatus(site, 'REPLICA')
+            .setReplicationStatus('REPLICA')
+            .setNonTransientLocation(undefined);
         return newEntry;
     }
 
+    _getGlobalReplicationStatus() {
```

Comment:
This LGTM, but why move the method here? I think it's helpful to have the `to...Entry` methods one after the next in this class 😸.

Reply:
I like when methods are close to where they are used, but no worry I can move it somewhere up in the class. The main point being making the function a class member :)

Reply:
done